addons/medical/XEH_postInit.sqf

```diff
@@ -174,52 +174,54 @@ GVAR(lastHeartBeatSound) = ACE_time;
         if ((ACE_time > GVAR(lastHeartBeat) + _interval)) then {
             GVAR(lastHeartBeat) = ACE_time;
 
-            // Pain effect
-            _strength = (_pain - (ACE_player getvariable [QGVAR(painSuppress), 0])) max 0;
-            _strength = _strength * (ACE_player getVariable [QGVAR(painCoefficient), GVAR(painCoefficient)]);
-            if (GVAR(painEffectType) == 1) then {
-                GVAR(effectPainCC) ppEffectEnable false;
-                if (_pain > (ACE_player getvariable [QGVAR(painSuppress), 0]) && {alive ACE_player}) then {
-                    _strength = _strength * 0.15;
-                    GVAR(effectPainCA) ppEffectEnable true;
-                    GVAR(effectPainCA) ppEffectAdjust [_strength, _strength, false];
-                    GVAR(effectPainCA) ppEffectCommit 0.01;
-                    [{
-                        GVAR(effectPainCA) ppEffectAdjust [(_this select 0), (_this select 0), false];
-                        GVAR(effectPainCA) ppEffectCommit (_this select 1);
-                    }, [_strength * 0.1, _interval * 0.2], _interval * 0.05, 0] call EFUNC(common,waitAndExecute);
-                    [{
-                        GVAR(effectPainCA) ppEffectAdjust [(_this select 0), (_this select 0), false];
+            // Pain effect, no pain effect in zeus camera
+            if (isNull (findDisplay 312)) then {
```

Comment:
First i wanted to use curatorCamera but i saw `if (!(isNull (findDisplay 312)) or !(alive ACE_player)) exitWith {` in ace_medical postinit so i used this.